src/daemon/rpc_command_executor.cpp

```diff
@@ -644,7 +644,7 @@ bool t_rpc_command_executor::print_connections() {
     }
   }
 
-  tools::msg_writer() << std::setw(30) << std::left << "Remote Host"
+  tools::msg_writer() << std::setw(74) << std::left << "Remote Host"
```

Comment:
Would it be possible to keep it at 30 unless an IPv6 peer is connected? It makes the `print_cn` list look weird now and 99% of users won't have any IPv6 connections.

Reply:
In fact, it's for i2p and onion addresses. They can be much longer than IPv6 addresses.
Is something relying on this field being fixed length? If not, we could make it variable length depending on the current peers' addresses.

Reply:
>Is something relying on this field being fixed length

Not sure, I did some testing with the following code and it seemed to work well.

```cpp
int width = std::any_of(res.connections.begin(), res.connections.end(), [](const auto &info) {
  return (epee::net_utils::address_type)info.address_type == epee::net_utils::address_type::tor ||
         (epee::net_utils::address_type)info.address_type == epee::net_utils::address_type::i2p ||
         (epee::net_utils::address_type)info.address_type == epee::net_utils::address_type::ipv6;
}) ? 74 : 30;
```

Reply:
I updated the patch to adapt the width of the host field to keep it as small as possible.  What do you think?